libs/telemetry/src/main/java/org/opensearch/telemetry/tracing/Span.java

```diff
@@ -93,4 +93,31 @@ public interface Span {
      */
     String getSpanId();
 
+    /**
+     * *
+     * @param key for which we need to look for value
+     * @return string attribute value
+     */
+    String getAttributeString(String key);
```

Comment:
Please revisit this strategy of having type specific getter.

Reply:
We need this as getting an attribute from ReadableSpan one should know the type of value stored in the attribute itself.